Signal/src/network/GiphyDownloader.swift

```diff
@@ -522,7 +585,7 @@ extension URLSessionTask {
             }
             guard !assetRequest.wasCancelled else {
                 // Discard the cancelled asset request and try again.
-                self.processRequestQueue()
+                self.removeAssetRequestFromQueue(assetRequest: assetRequest)
```

Comment:
Don't we still want to `processRequestQueue()` so we can try the next one?

Reply:
`removeAssetRequestFromQueue` calls `processRequestQueue`